src/app/login/login.component.ts

```diff
@@ -64,12 +68,36 @@ export class LoginComponent {
                 .then((data) => {
                     this.message = 'User created: ' + data.id.replace('org.couchdb.user:','');
                     this.reRoute();
+                }, (error) => {
+                    this.message = '';
+                    console.log('Need admin privilage to create a user');
+            });
+        } else {
+            this.message = 'Passwords do not match';
+        }
+    }
+
+    createAdmin({name,password,repeatPassword}:{name:string,password:string,repeatPassword:string}) {
+        if(password === repeatPassword) {
+            this.couchService.put('_node/nonode@nohost/_config/admins/' + name + ' -d ' + "'" + '"' + password + '"' + "'", {})

```

Comment:
The PUT request we send from the browser will differ from the `curl` requests in the CouchDB docs.  The first parameter should be the URL, which in this case is `'_node/nonode@nohost/_config/admins/' + name`, and the second parameter is the data sent, which in this case should just be the password.